danny/dannybs.css

```diff
@@ -0,0 +1,44 @@
+.container-fluid { background-color: #445B66;
```

Comment:
We won't be able to style things like `.container-fluid` as it will override other elements on the same page. Instead, for footer styling, let's use the `<footer>` tag and style that.